js/interface.js

```diff
@@ -2,6 +2,9 @@ var widgetId = Fliplet.Widget.getDefaultId();
 var data = Fliplet.Widget.getData(widgetId) || {};
 var appId = Fliplet.Env.get('appId');
 var headingValue = data.heading || 'Welcome to the login of this app';
+var canShowPassword = data.showPassword || false;
+
+$('.password-show-checkbox').prop('checked', canShowPassword);
```

Comment:
jQuery selector must be limited to within the widget instance.

Reply:
> jQuery selector must be limited to within the widget instance.

Can you please tell me how can I define the widget instance in the interface? Or is there some other way how it can be done?

Reply:
@skrupnyk That's my mistake. I thought this was `build.js`. All good 👍🏻